src/locales/de/ExampleQueries.json

```diff
@@ -37,7 +37,7 @@
         "Text": "Describe Query2"
       }
     },
-    "animalFeed": {
+    "feedSample": {
```

Comment:
Forgot to rename the other translation-objects?

Reply:
Done.